components/Usertable.js

```diff
@@ -0,0 +1,324 @@
+    inviteUserOpen: false,
+    addUserOpen: false,
+    columns: [
+      { title: "First Name", field: "first_name" },
+      { title: "Last Name", field: "last_name" },
+      { title: "Email", field: "email" },
+      {
+        title: "User Type",
+        field: "userType",
+        lookup: { 0: "Admin", 1: "Member" }
+      }
+    ],
+    data: [
+      {
+        first_name: "John",
+        last_name: "Smith",
+        email: "jsmith@gmail.com",
+        userType: 1
+      },
+      {
+        first_name: "Sam",
+        last_name: "Smith",
+        email: "jsmith@gmail.com",
+        userType: 0
+      },
+      {
+        first_name: "Bob",
+        last_name: "Smith",
+        email: "jsmith@gmail.com",
+        userType: 1
+      },
+      {
+        first_name: "Ben",
+        last_name: "Franklin",
+        email: "jsmith@gmail.com",
+        userType: 0
+      },
+      {
+        first_name: "Amy",
+        last_name: "Gutmann",
+        email: "jsmith@gmail.com",
+        userType: 0
+      },
+      {
+        first_name: "Rajiv",
+        last_name: "Gandhi",
+        email: "jsmith@gmail.com",
+        userType: 1
+      }
+    ]
+  });
+
+  const [values, setValues] = React.useState({
+    inputFirstName: "",
+    inputLastName: "",
+    inputEmail: "",
+    inputPassword: "",
+    userType: ""
+  });
+
+  const handleChange = event => {
+    setValues(oldValues => ({
+      ...oldValues,
+      [event.target.name]: event.target.value
+    }));
+  };
+
+  function handleInviteClick() {
+    setState(oldStates => ({
+      ...oldStates,
+      inviteUserOpen: true
+    }));
+  }
+
+  const handleInviteClose = () => {
+    setState(oldStates => ({
+      ...oldStates,
+      inviteUserOpen: false
+    }));
+  };
+
+  function handleAddClick() {
+    setState(oldStates => ({
+      ...oldStates,
+      addUserOpen: true
+    }));
+  }
+
+  const handleAddClose = () => {
+    setState(oldStates => ({
+      ...oldStates,
+      addUserOpen: false
+    }));
+
+    setValues({});
+  };
+
+  const handleAddSubmit = () => {
+    // Check if values are empty
+    // TODO: more sophisticated checking once connected to backend
+    if (
+      values.inputFirstName === "" ||
+      values.inputLastName === "" ||
+      values.inputEmail === "" ||
+      values.inputPassword === "" ||
+      values.userType === ""
+    ) {
+      return;
+    }
+
+    let newData = {
+      first_name: values.inputFirstName,
+      last_name: values.inputLastName,
+      email: values.inputEmail,
+      password: values.inputPassword,
+      userType: values.userType
+    };
+
+    let data = state.data;
+    data.unshift(newData);
+
+    console.log(data);
+    setState(oldStates => ({
+      ...oldStates,
+      addUserOpen: false,
+      data: data
+    }));
+
+    setValues({});
+  };
+
+  return (
+    <div>
+      <link
+        rel="stylesheet"
+        href="https://fonts.googleapis.com/icon?family=Material+Icons"
+      />
```

Comment:
It might be a good idea to move this `<link/>` to `Layout.js` instead, since then we don't have to import the fonts for each of our components!